iset.mm

```diff
@@ -82997,6 +83073,100 @@ Nonnegative integers (as a subset of complex numbers)
     $.
 
 
+$(
+=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=
+                              Extended nonnegative integers
+=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=
+
+  The function values of the hash (set size) function are either nonnegative
+  integers or positive infinity.  To avoid the need to distinguish between
+  finite and infinite sets (and therefore if the set size is a nonnegative
+  integer or positive infinity), it is useful to provide a definition of the
+  set of nonnegative integers extended by positive infinity, analogously to
+  the extension of the real numbers ` RR* ` , see ~ df-xr .
+
+$)
+
+  $c NN0* $. $( The set of extended nonnegative integers $)
+
+  $( The set of extended nonnegative integers. $)
+  cxnn0 $a class NN0* $.
+
+  $( Define the set of extended nonnegative integers that includes positive
+     infinity.  Analogue of the extension of the real numbers ` RR* ` , see
+     ~ df-xr .  (Contributed by AV, 10-Dec-2020.) $)
+  df-xnn0 $a |- NN0* = ( NN0 u. { +oo } ) $.
+
+  $( An extended nonnegative integer is either a standard nonnegative integer
```

Comment:
Here and below, I would remove "standard" since it makes us think of nonstandard models, which are not at play here.